app/controllers/api/v1/locations_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class LocationsController < ApplicationController
+      before_action :set_location, only: [:update, :destroy]
+
+      def create
+        @school = CodeSchool.find(params[:code_school_id])
+        @school.locations.build(location_params)
+        if @school.save
+          render json: @school
+        else
+          render json: { errors: @school.errors.full_messages }
+        end
+      end
+
+      def update
+        render json: @location.update(location_params) ? @location : { errors: @location.errors.full_messages }
```

Comment:
For readability for future contributors, can you use the `if/else` format here, instead of setting the `render` contents with a ternary.

Reply:
ok